CMakeLists.txt

```diff
@@ -472,16 +472,17 @@ function(unpack_db db_bzip2_file)
     STRING(REPLACE ".bz2" "" db_file "${db_bzip2_file}")
     find_program(UNZIPPER lbunzip2 bunzip2)
     if(EXISTS "${db_file}")
-        # message(WARNING "${db_file} already exists")
-    elseif(EXISTS "${db_bzip2_file}")
-        message(STATUS "${UNZIPPER} -k ${db_bzip2_file}")
+        message(STATUS "WARNING: ${db_file} already exists and will be overwritten")
```

Comment:
This situation is normal for local builds, but strange for CI.

Note: I use `message(STATUS "WARNING: ...)` because `message(WARNING...)` looks ugly and clutters up the screen.